### ISSUE TITLE
cannot import name 'TOKEN_TEXT'

### ISSUE THREAD

**Issue by ibrahimsamson** [phrase/django-phrase#19]
When loading ` {% load phrase_i18n %}` in my Django templates, These three errors occur. Which I think it is caused in the phrase.compact 
```
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/home/ibrahim/workspace//venv/lib/python3.6/site-packages/django/template/utils.py", line 66, in __getitem__
    return self._engines[alias]
KeyError: 'django'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/template/backends/django.py", line 121, in get_package_libraries
    module = import_module(entry[1])
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/phrase/templatetags/phrase_i18n.py", line 4, in <module>
    from django.template.base import TOKEN_TEXT, TOKEN_VAR
ImportError: cannot import name 'TOKEN_TEXT'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/utils/autoreload.py", line 54, in wrapper
    fn(*args, **kwargs)
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/channels/management/commands/runserver.py", line 69, in inner_run
    self.check(display_num_errors=True)
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/core/management/base.py", line 390, in check
    include_deployment_checks=include_deployment_checks,
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/core/management/base.py", line 377, in _run_checks
    return checks.run_checks(**kwargs)
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/core/checks/registry.py", line 72, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/contrib/admin/checks.py", line 79, in check_dependencies
    for engine in engines.all():
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/template/utils.py", line 90, in all
    return [self[alias] for alias in self]
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/template/utils.py", line 90, in <listcomp>
    return [self[alias] for alias in self]
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/template/utils.py", line 81, in __getitem__
    engine = engine_cls(params)
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/template/backends/django.py", line 25, in __init__
    options['libraries'] = self.get_templatetag_libraries(libraries)
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/template/backends/django.py", line 43, in get_templatetag_libraries
    libraries = get_installed_libraries()
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/template/backends/django.py", line 108, in get_installed_libraries
    for name in get_package_libraries(pkg):
  File "/home/ibrahim/workspace/venv/lib/python3.6/site-packages/django/template/backends/django.py", line 125, in get_package_libraries
    "trying to load '%s': %s" % (entry[1], e)
django.template.library.InvalidTemplateLibrary: Invalid template library specified. ImportError raised when trying to load 'phrase.templatetags.phrase_i18n': cannot import name 'TOKEN_TEXT'

```

**Comment by rlisowski** [comment]
@ibrahimsamson thank you for reporting this, we will take a look shortly.

**Comment by Xivolkar** [comment]
@rlisowski I'm hitting this issue now as well while performing a `django` upgrade to 2.2 (LTS). I'm using `django-phrase` - `1.0.7`

**Comment by theSoenke** [comment]
Hey @Xivolkar, sadly we did not get to this yet. But I'll raise this again internally and we'll try to get a fix out as soon as possible

**Comment by Xivolkar** [comment]
@theSoenke So it looks like this problem may only be in the release packages ([1.0.7](https://pypi.org/project/django-phrase/1.0.7/)), which is, it doesn't seem to ship the latest bits. For example, the fix to this issue is present in `compat.py`.

So I'm currently using a workaround which is to insert this in the dependencies instead

`-e git+https://github.com/phrase/django-phrase@806d644cb9406e150db003fd44cbb4c3202522ae#egg=django-phrase`

@docstun @rlisowski

**Comment by kelkarajay** [comment]
Moin! @theSoenke @docstun @rlisowski Any updates on this? We'd like to go with a concrete release tag rather than a git commit hash ;)

**Comment by theSoenke** [comment]
@kelkarajay That's a good point :) We'll try to get a new release out this week

**Comment by theSoenke** [comment]
@kelkarajay We just released a new version 1.0.8 which should include this fix
